pip/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
-name = "qsharp"
-version = "0.0.0"
+name = "qsharp_python"
```

Comment:
I don't think it's great to have "python" in the package name since clearly this is a Python package.

Also, apparently the naming conventions advise against using underscores in the package name. (See: https://peps.python.org/pep-0008/#package-and-module-names)

I suppose now is the time to close on a new name. `qsharppreview`? `qsharpnew`? @billti ?

Reply:
OK, "qsharppreview" candidate makes sense to me then.

Reply:
'qsharp-preview' would be preferred and in line with Python package naming.

Reply:
@billti, Are we OK replacing 
`qsharp_jupyterlab` with 
`qsharp-jupyterlab`?
(replacing underscore with dash)

Reply:
(FYI, for it to not be a surprise)
If the package name contains dashes, the .whl file name still contains the underscores for the package name part. E.g. the [package](https://pypi.org/project/django-cricket-statistics/#files) name is 
`django-cricket-statistics` (with dashes), but the .whl file is 
`django_cricket_statistics-0.0.1-py3-none-any.whl` (with underscores in the fragment "django_cricket_statistics").

Reply:
Updated the package names to 
qsharp-preview
qsharp-jupyterlab